conda-recipes/sentencepiece-feedstock/recipe/build.sh

```diff
@@ -17,6 +17,47 @@
 
 set -ex
 
+build_tf_wrapper() {
+  if [ "$1" != "" ]; then
+    pkg_name="==$1"
+  fi
+
+  export IBM_POWERAI_LICENSE_ACCEPT=yes
+  conda create -n tf$1 -y python=3.6
+
+  conda activate tf$1
+  conda config --prepend channels https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+  conda config --prepend channels http://ausgsa.ibm.com/projects/p/powerai-conda/conda-early-access/
```

Comment:
We can't reference GSA in external github. Instead use: https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda-early-access/

Reply:
Yeah. I've removed that. Thanks Billy!